torch/_higher_order_ops/scan.py

```diff
@@ -49,7 +49,6 @@ def scan(
     ],
     init: pytree.PyTree,
     xs: pytree.PyTree,
-    /,
```

Comment:
* is pobably good enough

Reply:
Are you sure you want to remove this? It's easy to remove it, it's difficult to put it back :P.